Model/Adapters/NestedArrayAdapter.php

```diff
@@ -1,46 +1,37 @@
 <?php
 /**
- * *
- *  * Copyright © Elias Kotlyar - All rights reserved.
- *  * See LICENSE.md bundled with this module for license details.
- *
+ * Copyright © 2016 FireGento e.V. - All rights reserved.
+ * See LICENSE.md bundled with this module for license details.
  */
+
 namespace FireGento\FastSimpleImport\Model\Adapters;
 
 class NestedArrayAdapter extends ArrayAdapter
 {
-    protected $multipleValueSeparator;
+    private string $multipleValueSeparator;
 
-    /**
-     * ArrayAdapter constructor.
-     * @param array $data
-     * @param string $multipleValueSeparator
-     */
-    public function __construct($data, $multipleValueSeparator = ', ')
+    public function __construct(array $data, string $multipleValueSeparator = ', ')
     {
         $this->multipleValueSeparator = $multipleValueSeparator;
 
         parent::__construct($data);
-        foreach ($this->_array as &$row) {
-            foreach ($row as $colName => &$value)
+
+        foreach ($this->array as &$row) {
+            foreach ($row as &$value) {
                 if (is_array($value)) {
                     $this->convertToArray($value);
                 }
+            }
         }
-        //print_r($this->_array);
-
     }
 
     /**
      * Transform nested array to string
-     *
-     * @param array $line
-     * @return void
      */
-    protected function convertToArray(&$line)
+    private function convertToArray(array &$line)
```

Comment:
After removal of the annotation a return type would be appropriate.

Reply:
I added `:void` and a more detailed comment.